### PR TITLE
Allow users to choose a time in the next 24 hours to send a file

### PR DESCRIPTION
**Pull request by quis**
schedule-xx

image

image

### CODE REVIEW

**Comment by minglis** [comment]
Guess this depends on: https://github.com/alphagov/notifications-api/pull/635

File: app/main/views/jobs.py [hunk 1]
```diff
@@ -65,7 +65,10 @@ def _set_status_filters(filter_args):
 def view_jobs(service_id):
     return render_template(
         'views/jobs/jobs.html',
-        jobs=add_rate_to_jobs(job_api_client.get_job(service_id)['data'])
+        jobs=filter(
+            lambda job: job['job_status'] != 'scheduled',
+            add_rate_to_jobs(job_api_client.get_job(service_id)['data'])
+        )
```
Comment (leohemsted) [inline app/main/views/jobs.py]: filter's pretty oldschool - the more pythonic way is a comprehension such as

```python
jobs = add_rate_to_jobs(job_api_client.get_job(service_id)['data'])
return render_template(
    'views/jobs/jobs.html',
    jobs=[job for job in jobs if job['job_status'] != 'scheduled']
)
```

File: app/templates/components/radios.html [hunk 2]
```diff
@@ -49,6 +83,7 @@
           />
           {% if branding_dict.get(value, {}).get('colour') %}
             <span style="background: {{ branding_dict[value].colour }}; display: inline-block; width: 3px; height: 27px"></span>
+        <div class="radio-select-column">
```
Comment (leohemsted) [inline app/templates/components/radios.html]: weird indentation and lack of closing tag - is this a copy + paste error?

File: app/main/views/dashboard.py [hunk 3]
```diff
@@ -116,9 +116,21 @@ def get_dashboard_partials(service_id):
...
         job_api_client.get_job(service_id, limit_days=7)['data']
     ))
+    scheduled_jobs = filter(
+        lambda job: job['job_status'] == 'scheduled',
+        jobs
+    )
+    immediate_jobs = filter(
+        lambda job: job['job_status'] != 'scheduled',
+        jobs
+    )
```
Comment (leohemsted) [inline app/main/views/dashboard.py]: these should also be comprehensions rather than filters

File: app/main/views/dashboard.py [hunk 4]
```diff
@@ -116,9 +116,21 @@ def get_dashboard_partials(service_id):
         lambda job: job['original_file_name'] != current_app.config['TEST_MESSAGE_FILENAME'],
         job_api_client.get_job(service_id, limit_days=7)['data']
     ))
+    scheduled_jobs = filter(
```
Comment (leohemsted) [inline app/main/views/dashboard.py]: scheduled jobs should be sorted, probably by `asc scheduled_for` time, but i'll leave ordering decision to you - either way it's currently reverse `created_at`

Comment (quis) [inline app/main/views/dashboard.py]: Sorted by `scheduled_for` in f573678

File: app/main/views/dashboard.py (hunk 3 above)
Comment (quis) [inline app/main/views/dashboard.py]: lambdas removed in 	c2839ce

File: app/main/views/jobs.py (hunk 1 above)
Comment (quis) [inline app/main/views/jobs.py]: lambdas removed in 	c2839ce

File: app/templates/components/radios.html (hunk 2 above)
Comment (quis) [inline app/templates/components/radios.html]: Good spot. Was a bad merge resolution. Have squashed it out.

**Comment by leohemsted** [comment]
checked out & tested locally, looks good 👍

**Pull request merged**